File: src/main/kotlin/ejerciciosJava/Usuario.java
package ejerciciosJava;

import java.util.List;

public class Usuario {
    String nombre;
    List<Videojuego> inventario;

    public Usuario(String n, List<Videojuego> i) {
        this.nombre = n;
        this.inventario = i;
    }

    public void agregarJuego(Videojuego juego) {
        inventario.add(juego);
    }

    public void eliminarJuego(Videojuego juego) {
        inventario.remove(juego);
    }

    public void mostrarJuego() {
        inventario.forEach(System.out::println);
    }

}

File: src/main/kotlin/ejerciciosKotlin/Usuario.kt
package org.example.ejerciciosKotlin

class Usuario(val nombre:String, val videojuegos:MutableList<Videojuego>) {

    fun agregarJuego(juego:Videojuego) {
        videojuegos.add(juego)
    }

    fun eliminarJuego(videojuego: Videojuego) {
        videojuegos.remove(videojuego)
    }

    fun mostrarJuego() {
        videojuegos.forEach {
            println(it)
        }
    }
}